tests/justice_board.test.ts

```diff
@@ -0,0 +1,80 @@
+import request from 'supertest';
+import { Express } from 'express';
+import { Collection } from 'mongodb';
+
+import { getServer } from '../index';
+import { Duty } from '../schemas/duty.zschema';
```

Comment:
Not a big issue, but is this with a caps D because its a type? Because this is a schema not a type... For me its most important to be coherent. If all your schemas are with Caps at the start that's ok, only think whether you might prefer to actually write schemas with small letters, and types with capital letters in order to differentiate more easily

Reply:
It is a type, the schema is DutySchema